test-requirements.txt

```diff
@@ -3,3 +3,4 @@ flake8
 pytest
 pytest-cov
 pylint
+cloudpickle==1.5.0
```

Comment:
We shouldn't need this if we put it into `setup.py`

Reply:
Removed